### ISSUE TITLE
[BUG]:

### ISSUE THREAD

**Issue by misspastelwitch** [fdnd-agency/nebulaxplorer#108]
### Description

When trying to visit the news page on mobile, the menu opens automatically, and cannot be shut. 

### Expected behavior

The menu shuts when tapping on the menu icon

### Actual behavior

The menu does not shut, and tapping on any page names automatically brings you to the home page.

### What device are you experiencing the problem on?

Mobile

### What operating system are you experiencing the problem on?

iOS

### What browsers are you experiencing the problem on?

Other
